examples/plot_bootstrapped_coffecients_for_lm.py

```diff
@@ -0,0 +1,144 @@
+"""
+================================================================
+Plot bootstrapped beta coefficients for a linear model estimator
+================================================================
+
+"""
+
+# Authors: Jose C. Garcia Alanis <alanis.jcg@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from sklearn.linear_model import LinearRegression
```

Comment:
Can you maybe time this vs. Ridge (with alpha=0)?

Reply:
I think the backbone is different, and that Ridge with Cholesky should be fastest for multiple targets.